### DISCUSSION TITLE
How to set the `align` parameter for MonadTall layout ?

### DISCUSSION THREAD

**Discussion by lyndhurst** [qtile/qtile#4683] [Q&A] [answered]
Hi,
I am setting up my first qtile config, and so far I managed to do everything thanks to the extensive doc that comes with it (and a lot of trial and errors!).

I am trying to fine tune the MonadTall layout, and I want to control the `adjust` option so that secondary clients always open to the right.

The doc says it can be one of two values: `MonadTall._left` or `MonadTall._right`. I do not understand if this is supposed to be a string, a python object, a function...

I tried all the following to no avail. Strings do not throw errors but have no effect on new clients, non string attempts trow errors. 
Of course I tried them one at a time, and every time a string containing `left` would not throw an error, I tried with `right` to see if it had any effect. I also tried integers since the doc states that the default value is `0`.

```python
        layout.MonadTall(
            align = MonadTall._left,
            align = self._left,
            align = _left,
            align = left,
            align = "MonadTall._left",
            align = "self._left",
            align = "_left",
            align = "left",
            align = 1,
            align = 2,
            align = "1",
            align = "2",
            ),
```

 Thanks for your help.

**Comment by elParaguayo** [comment, marked as answer]
`layout.MonadTall._left`

**Comment by lyndhurst** [comment]
Thanks a lot for your quick answer. It works as expected now:

```python
        layout.MonadTall(
            align = layout.MonadTall._left,
            )
```

**Reply by elParaguayo** [reply]
Sorry for the shorter reply earlier. `_left` is a class attribute of the `MonadTall` class so you access it however you're importing `MonadTall`.

**Reply by lyndhurst** [reply]
Thanks for the clarification, it makes more sense to me now.
